c2corg_ui/templates/waypoint/edit.html

```diff
@@ -90,7 +90,7 @@ <h2 class="heading show-phone"><span translate>title</span></h2>
                  ng-class="{ 'has-error': editForm.lang.$touched && editForm.lang.$invalid, 'has-success': editForm.lang.$valid}">
               <label><span translate>lang</span> *</label>
               % if waypoint_lang:
-              <input class="form-control" disabled x-translate value="${waypoint_lang}">
+              <input class="form-control" disabled value="{{mainCtrl.translate('${waypoint_lang}')}}">
```

Comment:
It seems the extractor is smart enough to extract only strings, not variables.
For instance ``{{'foo' | translate}}`` will make the string "foo" added to the POT file.
On the other hand ``{{foo | translate}}`` won't.